sdk/java-sdk-spring/src/test/scala/kalix/javasdk/impl/ActionDescriptorFactorySpec.scala

```diff
@@ -417,6 +420,22 @@ class ActionDescriptorFactorySpec extends AnyWordSpec with ComponentDescriptorSu
       }.getMessage should include("You cannot use @Subscribe.EventSourcedEntity annotation in both methods and class.")
     }
 
+    //TODO remove ignore after updating to Scala 2.13.11 (https://github.com/scala/scala/pull/10105)
+    "validates if there are missing event handlers for event sourced Entity Subscription at type level" ignore {
+      intercept[InvalidComponentException] {
+        Validations.validate(classOf[MissingHandlersWhenSubscribeToEventSourcedEntityAction]).failIfInvalid
+      }.getMessage should include(
+        "Missing event handler for kalix.spring.testmodels.eventsourcedentity.EmployeeEvent$EmployeeEmailUpdated")
+    }
+
+    //TODO remove ignore after updating to Scala 2.13.11 (https://github.com/scala/scala/pull/10105)
+    "validates if there are missing event handlers for event sourced Entity Subscription at method level" ignore {
+      intercept[InvalidComponentException] {
+        Validations.validate(classOf[MissingHandlersWhenSubscribeToEventSourcedOnMethodLevelEntityAction]).failIfInvalid
+      }.getMessage should include(
+        "Missing EmployeeEntity event handler for kalix.spring.testmodels.eventsourcedentity.EmployeeEvent$EmployeeEmailUpdated")
```

Comment:
Because we are not running the tests, we didn't catch that the test is wrong.

The message will be: 'Component EmployeeEntity missing event handler for...' instead.

But I think if should even be:

"Component 'EmployeeEntity' is missing event handler for 'Event'"

Note the single quotes and the `is` before "missing".

Reply:
Yeah, it sucks, but haven't figured out any workaround.